### DISCUSSION TITLE
Issue for Wan access

### DISCUSSION THREAD

**Discussion by Narkowii** [TandoorRecipes/recipes#2984] [Q&A]
Hello, after some modification, i can acces Tandoor Recipes with wan by https.
(my setup : Proxmox LXC)

But i don't see recipes, and i can't create recipes....I don't know why...

<img width="608" alt="Annotation 2024-02-27 102200" src="https://github.com/TandoorRecipes/recipes/assets/153772234/d9051bee-523a-4275-956f-ce02cbf21ea3">

**Comment by smilerz** [comment]
What errors are in the browser console?

**Comment by Narkowii** [comment]
I find it....in the nginx conf...

i change this ` proxy_set_header X-Forwarded-Proto $scheme;` by ` proxy_set_header X-Forwarded-Proto 'https';`
